src/components/tables/test/utils.test.ts

```diff
@@ -0,0 +1,40 @@
+import { dateSorter, renderActivities, renderSiteIdAsLink } from '../utils';
+import { render, screen } from '@testing-library/react';
+
+test('dateSorter', () => {
```

Comment:
🤔 This is the sort of function that a property test would be perfect for. If you're curious check out the fast-check package. Or take CS2800

Reply:
Agree, but I'd rather add this + improve other existing tests in a separate ticket. Left a todo.